src/main/scala/org/allenai/wikitables/TestWikiTablesCli.scala

```diff
@@ -60,7 +55,7 @@ class TestWikiTablesCli extends AbstractCli() {
 
     // Read test data.
     val testData = options.valuesOf(testDataOpt).asScala.flatMap(filename => {
-      WikiTablesUtil.loadDataset(filename, true, options.valueOf(derivationsPathOpt), -1)
+      WikiTablesUtil.loadDataset(filename, false, null, -1)
```

Comment:
I would try hard to avoid using `null` in scala code.  I'd prefer changing the type of this argument in `loadDataset` to `Option[String]`, then when it's passed to the java code that needs to use null, do `derivationsPath.getOrElse(null)`, so that our use of `null` in scala is limited to the interface with java.